src/Featurizers/LagLeadOperatorFeaturizer.h

```diff
@@ -0,0 +1,237 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#pragma once
+
+#include "../Featurizer.h"
+#include "../Archive.h"
+#include "Components/PipelineExecutionEstimatorImpl.h"
+#include "Components/FilterDecoratorFeaturizer.h"
+#include "Components/GrainFeaturizerImpl.h"
+#include "Components/WindowFeaturizerBase.h"
+
+namespace Microsoft {
+namespace Featurizer {
+namespace Featurizers {
+
+/////////////////////////////////////////////////////////////////////////
+///  \class         LagLeadOperatorTransformer
+///  \brief         Copy values from prior data or future data
+///                 Input type is a tuple of std::vector<std::string>, representing grains, and std::double_t, representing target column
+///
+template <typename InputTupleT>
```

Comment:
This transformer isn't grain aware right? Isn't the grain wrapper going to do that for us? So isn't this type not a tuple and just the target value?

Reply:
Oh I think you are right. The input type should just be the target value type.